test/integration/features/cmd-openshift.feature

```diff
@@ -48,7 +48,7 @@ cluster in VM provided by Minishift.
 
   Scenario: User deploys nodejs example application from OpenShift repository
     Given Minishift has state "Running"
-     When executing "oc new-app https://github.com/sclorg/nodejs-ex -l name=myapp" succeeds
+     When executing "oc new-app https://github.com/sclorg/nodejs-ex -l name=myapp" retrying 20 times with wait period of "3s"
```

Comment:
Interesting, I did not catch this when I went through the integration test with respect to this issue.

Reply:
@LalatenduMohanty I have removed it previously as I have added docker container check before this step, tried this solution now again, but it does not help: https://ci.centos.org/job/minishift-pr/5535/consoleFull. So figuring out other solution.